src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -19,6 +19,7 @@
 import re
 import shutil
 import time
+import glob
```

Comment:
The usings are in alphabetical order. What's line 22 should be the new line 17 pushing everything down.